client/src/routes/SessionModal/SessionModal.tsx

```diff
@@ -122,11 +165,18 @@ const SessionModal = () => {
       'Session Type': session.type,
       'Session Start Time': session.startTime,
     });
-  };
+  }, [
+    session.startTime,
```

Comment:
If sessions does not change much using the whole object as dependency is cleaner IMHO

Reply:
hmmm... not sure, like if we set the session state to a new object but it is still the same session to a large extent (fields we might depend upon didn't change), it will trigger unnecessary code, no?